src/server/plugins/engine/components/AutocompleteField.test.ts

```diff
@@ -132,7 +132,7 @@ describe.each([
 
         expect(result.error).toEqual(
           expect.objectContaining({
-            message: expect.stringContaining(`Select ${label}`)
+            message: `Select ${label}`
```

Comment:
See PR https://github.com/DEFRA/forms-runner/pull/445 to fix "Select…" → "Enter…" for Autocomplete